### ISSUE TITLE
ubuntu-1004-32 tests resurrected

### ISSUE THREAD

**Issue by seancribbs** [basho/giddyup#105]
The seed occasionally repopulates tests for ubuntu-1004-32 but these are not valid beyond Riak 1.2 (please confirm @jaredmorrow). The seed should not create them.

```
$ foreman run bin/rake db:seed
Creating test verify_secondary_index_reformat with {"platform"=>"ubuntu-1004-32", "min_version"=>"1.3.1", "backend"=>"eleveldb", "max_version"=>"1.4.99"} for projects ["riak", "riak_ee"]
Creating test replication2_rt_sink_connection with {"platform"=>"ubuntu-1004-32", "min_version"=>"1.4.4", "max_version"=>"1.4.99"} for projects ["riak", "riak_ee"]
Creating test replication2_connections with {"platform"=>"ubuntu-1004-32", "min_version"=>"1.4.4", "max_version"=>"1.4.99"} for projects ["riak", "riak_ee"]
Creating test repl_aae_fullsync with {"platform"=>"ubuntu-1004-32", "min_version"=>"1.4.4", "max_version"=>"1.4.99"} for projects ["riak", "riak_ee"]
```

**Comment by jaredmorrow** [comment]
I'd be fine if 32bit was completely removed for all versions since it is outside of our (2 major version) support as well.

**Comment by seancribbs** [comment]
Fixed in b29ff97
